common/tmx_level.go

```diff
@@ -139,7 +132,13 @@ func createLevelFromTmx(r Resource) (*Level, error) {
 
 	// Load in the images needed for the tilesets
 	for k, ts := range tlvl.Tilesets {
-		ts.Image = Files.Image(path.Base(ts.ImageSrc.Source))
+		url := path.Join(path.Dir(tmxUrl), ts.ImageSrc.Source)
+		if err := engo.Files.Load(url); err != nil {
+			return lvl, err
+		}
+		image, _ := engo.Files.Resource(url)
+		texResource := image.(TextureResource)
```

Comment:
This will `panic` on failure. Better use

```go
texResource, ok := image.(TextureResource)
if !ok {
    return nil, fmt.Sprintf("resource is not of type `TextureResource`: %q", url)
}
```